test/system/252-quadlet.bats

```diff
@@ -523,6 +524,7 @@ ContainerName=$NAME
 Image=$IMAGE
 Secret=$SECRET_NAME,type=env,target=MYSECRET
 Exec=sh -c "echo STARTED CONTAINER; echo "READY=1" | socat -u STDIN unix-sendto:\$NOTIFY_SOCKET; top"
+Notify=yes
```

Comment:
Shouldn't this be `=true`?

Reply:
quadlet supports true, yes or 1 for booleans. I think yes is common for systemd units settings.

Reply:
Yes, the convention in systemd unit files is to use `yes` or `no`